portfolio/src/main/webapp/index.html

```diff
@@ -83,8 +83,12 @@ <h3>This is a picture of a seagull.</h3>
                     <label>Have any questions or comments? </label>
                     <input type="text" name ="user-comment" placeholder="Submit them here!">
                     <input type = "submit" name = "submit" value ="submit">
+                    <br>
+                    <label>How many previous comments would you like displayed?</label>
+                    <input type="number" name="comment-number" min ="0" value = "5">
```

Comment:
this value isn't dynamic, AFAIK. If I change this value, the number of comments being displayed doesn't change.

Reply:
On my end this value works. You do have to hit the submit button before the number of comments displayed is changed, though (it's part of the comment form). I can move the submit button so that makes more sense. Does it work if you try that?

Reply:
ahh okay I see. So you can only change that value if you also submit a comment (or don't submit a comment, but that'll still submit a blank comment). I just tested that out and I can see it works that way so that's good. I would definitely recommend having two different functions here though. There should be a button for submitting a comment and a button for changing the number of comments displayed. You'll likely have two functions as well, one to save off a new comment and one to send the comments to display back to the client. I'll go into more detail about that in the comment on the java file.